rails/app/policies/portal/permission_form_policy.rb

```diff
@@ -3,21 +3,16 @@ class Portal::PermissionFormPolicy < ApplicationPolicy
   class Scope < Scope
     def resolve
       if user.has_role?('admin')
-        all
+        scope.all
       elsif user.is_project_admin? || user.is_project_researcher?
-        where = []
-        params = {}
-        if user.is_project_admin?
-          where << "(project_id in (:admin_project_ids))"
-          params[:admin_project_ids] = user.admin_for_projects.map { |p| p.id }
-        end
-        if user.is_project_researcher?
-          where << "(project_id in (:researcher_project_ids))"
-          params[:researcher_project_ids] = user.researcher_for_projects.map { |p| p.id }
-        end
-        scope.where([where.join(" OR "), params])
+        admin_project_ids = user.admin_for_projects.pluck(:id)
```

Comment:
This went from 1 query to 3 queries.  I don't think it will be a problem as those tables are small but just wanted point that out.

Same for `management_policy_scope` in the other file.

Reply:
Thanks @dougmartin, that's a good point. I replaced .pluck with .select and combined both ID sets via OR condition:
https://github.com/concord-consortium/rigse/pull/1343/commits/57151913401abc9c945770116ed7bd899c52038f
I guess it's going to be executed as one query now. Let me know if you think it's not true.